src/kv/generic_serialise_wrapper.h

```diff
@@ -50,6 +50,7 @@ namespace kv
     W private_writer;
     W* current_writer;
     Version version;
+    Term term; // Used for encryption IV only
```

Comment:
Taking a TxID down that path may end up being a little cleaner.

Reply:
Done.